.github/workflows/bitcoin-tests.yml

```diff
@@ -1,205 +1,173 @@
-##
-## Bitcoin Integration Tests
-##
+## Github workflow to run bitcoin tests
```

Comment:
This new workflow is far simpler and easier to read than what was here before - 10/10